metrics/density/fast_footprint.sh

```diff
@@ -420,7 +322,8 @@ launch_containers() {
 	done
 
 	for n in $(seq 1 $leftovers); do
-		docker run --rm -d --runtime=$RUNTIME $PAYLOAD_RUNTIME_ARGS $PAYLOAD $PAYLOAD_ARGS &
+		containers+=($(random_name))
+		sudo ctr run --memory-limit $PAYLOAD_RUNTIME_ARGS -d --runtime=$CONTAINERD_RUNTIME $PAYLOAD ${containers[-1]} sh -c $PAYLOAD_ARGS &
```

Comment:
move `--memory-limit` to the `PAYLOAD_RUNTIME_ARGS` value?

Reply:
@chavafg done